MAUI/ImageEditor/crop.md

```diff
@@ -104,7 +104,7 @@ imageEditor.Crop(ImageCropType.Circle);
 

```

Comment:
<img width="367" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/92788387/64a14684-b08a-4a06-b3c0-7a7b4f211499">

Update content:

This means that no aspect ratio and the cropped image retains its original width and height ratio.